bindings/ios/MEGASdk.mm

```diff
@@ -1786,6 +1789,30 @@ - (MEGANodeList *)nodeListSearchForNode:(MEGANode *)node searchString:(NSString
     return [[MEGANodeList alloc] initWithNodeList:self.megaApi->search((node != nil) ? [node getCPtr] : NULL, (searchString != nil) ? [searchString UTF8String] : NULL, YES) cMemoryOwn:YES];
 }
 
+- (NSMutableArray *)recentActions {
+    MegaRecentActionBucketList *megaRecentActionBucketList = self.megaApi->getRecentActions();
+    NSUInteger count = (NSUInteger)megaRecentActionBucketList->size();
+    NSMutableArray *recentActionBucketMutableArray = NSMutableArray.alloc.init;
```

Comment:
It would be better use `initWithCapacity` because we know the size of the bucket list (`count`)

Reply:
As discussed we can save some casting by using an int for the variable `count`, and reuse it on the loop.

Reply:
And only do the casting on the initWithCapacity 🙂

Reply:
sounds like a good optimisation, please do make that change.   I think we can merge first though as it will work correctly as is.  cheers